### PR TITLE
Increase patience for ReceptionistExampleSpec

### PR DESCRIPTION
**Pull request by chbatey**
This test doesn't extend AkkaSpec doesn't pick up the larger patience
for other tests

### CODE REVIEW

**Review by johanandren — APPROVED** [review]
Looks good and very patient to me.

File: akka-cluster-typed/src/test/scala/docs/akka/cluster/typed/ReceptionistExampleSpec.scala [hunk 1]
```diff
@@ -194,6 +195,9 @@ class ReceptionistExampleSpec extends WordSpec with ScalaFutures {
   import ReceptionistExampleSpec._
   import PingPongExample._
 
+  implicit override val patienceConfig =
+    PatienceConfig(timeout = scaled(Span(10, Seconds)), interval = scaled(Span(100, Millis)))
+
   "A local basic example" must {
     "show register" in {
       val system = ActorSystem[Nothing](guardian, "PingPongExample")
```
[the remark this answers is not in the document]
Comment (chbatey) [reply akka-cluster-typed/src/test/scala/docs/akka/cluster/typed/ReceptionistExampleSpec.scala]: They don't shut down and are also not used in the docs so may remove these for now

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by patriknw** [comment]
Refs https://github.com/akka/akka/issues/26127

**Pull request merged**